### DISCUSSION TITLE
Using Steam Deck Trackpads

### DISCUSSION THREAD

**Discussion by sagewitch** [ihhub/fheroes2#6373] [Q&A] [answered]
I tried using the trackpads in game mode on Steam Deck but it doesn't seem to be working. Is there a way to enable this? It works great on desktop mode but I prefer game mode as it's quicker to start up and I can play it in fullscreen. Was also hoping to play it on the go without the need for a mouse. Thanks

**Comment by zenseii** [comment]
Hi, @sagewitch. I think we need to ask someone who has made the adaptations for Steam Deck. I believe @Laserlicht made the Flatpak package which is catered to Steam Deck?

**Comment by Laserlicht** [comment, marked as answer]
It works without problems even in gamemode. However, you have to manually adjust the controls for non-steam games (how would steam know that?).
`(In-Game) Steam Button -> Game -> Controller-Settings`

In my case, I found the "Keyboard (WASD) and Mouse" option quite good.

Then the triggers are right and left mouse button and the right trackpad the mouse itself.

But I would wait until the next release. Then the adventuremap can also be dragged and you no longer have to use the (on the steam deck) annoying screen edges to scroll.

**Reply by zenseii** [reply]
Thanks for the reply and answer!

**Reply by sagewitch** [reply]
It works great now! Strange, I usually check the settings with other controllers I use, not sure why I didn't this time. Thanks
